src/typescript-generator/generate.js

```diff
@@ -54,10 +47,12 @@ async function getPathsFromSpecification(specification) {
   }
 }
 
-// eslint-disable-next-line max-statements
+// eslint-disable-next-line max-statements, max-params
 export async function generate(
   source,
   destination,
+  routes,
+  types,
```

Comment:
I would suggest collapsing these into a single argument. It will make the calling code simpler and more readable, and you can use destructuring to avoid extra verbosity. (`generate(source, destination, { routes, types })`);

Reply:
done